server/src/main/java/com/cloud/resourcelimit/ResourceLimitManagerImpl.java

```diff
@@ -972,11 +980,59 @@ public long countCpusForAccount(long accountId) {
         for (UserVmJoinVO vm : userVms) {
             cputotal += Long.valueOf(vm.getCpu());
         }
-        return cputotal;
+
+        ServiceOffering defaultRouterOffering = null;
+        String globalRouterOffering = VirtualNetworkApplianceManager.VirtualRouterServiceOffering.value();
+        if (globalRouterOffering != null) {
+            defaultRouterOffering = _serviceOfferingDao.findByUuid(globalRouterOffering);
+        }
+        if (defaultRouterOffering == null) {
+            defaultRouterOffering =  _serviceOfferingDao.findByName(ServiceOffering.routerDefaultOffUniqueName);
+        }
```

Comment:
```suggestion
        ServiceOffering defaultRouterOffering = getDefaultServiceOfferingOrSomeOtherNewName();
```

this happens below as well, for memory.